### PR TITLE
Maquetado footer

### PR DESCRIPTION
**Pull request by andyOlarte514**
# Summary
- add the code of the footer component to the `index.html`
- create archive `footer.scss` and import to `application.scss`

# Trello Card
- https://trello.com/c/QvP9b2Em

# Screenshots
image

### CODE REVIEW

File: training-maquetado/index.html [hunk 1]
```diff
@@ -133,5 +141,14 @@
       </div>
     </article>
   </section>
+  <footer class="footer d-flex align-center justify-center fd-column">
+    <h3>we are game changers</h3>
+    <div class="footer-logo">
+      <img src="./assets/wolox_logo_white.svg" alt="Wolox Logo">
```
Comment (CarlosHWolox) [inline training-maquetado/index.html]: maybe use the class aswell directly on the img

File: training-maquetado/scss/content-layout.scss [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-    padding-bottom: 0;
-    margin-bottom: 50px;
+  &:first-of-type {
+    margin: 140px 0 50px;
   }
 
-  &:last-child {
-    padding-top: 0;
-    margin-top: 50px;
+  &:last-of-type{
```
Comment (CarlosHWolox) [inline training-maquetado/scss/content-layout.scss]: space

File: training-maquetado/index.html [hunk 3]
```diff
@@ -133,5 +138,12 @@
       </div>
     </article>
   </section>
+  <footer class="footer d-flex align-center justify-center fd-column">
+    <h3>we are game changers</h3>
+    <img class="footer-logo" src="./assets/wolox_logo_white.svg" alt="Wolox Logo">
+    <small>
```
Comment (CarlosHWolox) [inline training-maquetado/index.html]: I would add this a class as if for some reason in the future you would need to add another small element you would need to change the style of this one.

File: training-maquetado/index.html [hunk 4]
```diff
@@ -146,5 +146,12 @@ <h2 class="tech-desc-img">Release: 2005</h2>
       </div>
     </article>
   </section>
+  <footer class="footer d-flex align-center justify-center fd-column">
+    <h3 class="footer-title">we are game changers</h3>
+    <img class="footer-logo" src="./assets/wolox_logo_white.svg" alt="Wolox Logo">
+    <small class="footer-copyright">
```
Comment (braiandickson) [inline training-maquetado/index.html]: I would use a span tag here.

File: training-maquetado/scss/footer.scss [hunk 5]
```diff
@@ -0,0 +1,30 @@
+.footer {
+  width: 100%;
+  height: 250px;
+  max-height: 100%;
+  background-color: $blue;
+  position: relative;
```
Comment (braiandickson) [inline training-maquetado/scss/footer.scss]: You can use flexbox for this.

File: training-maquetado/scss/footer.scss [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    font-size: 24px;
+  }
+
+  .footer-logo {
+    width: 100%;
+    max-width: 400px;
+  }
+
+  .footer-copyright {
+    position: absolute;
```
Comment (braiandickson) [inline training-maquetado/scss/footer.scss]: The same here.

File: training-maquetado/scss/footer.scss [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    bottom: 10%;
```
Comment (braiandickson) [inline training-maquetado/scss/footer.scss]: Then this prop isn't required. And the next too.

**Review by braiandickson — CHANGES_REQUESTED** [review]

**Review by braiandickson — APPROVED** [review]

**Pull request merged**